### PR TITLE
Fixed First Japanese UHV text for all languages and speeds

### PR DESCRIPTION
**Pull request by Imperator-Knoedel**
"without ever losing a city" was only displayed for normal speed, not Epic and Marathon

### CODE REVIEW

File: Assets/XML/Text/Victory.xml [hunk 1]
```diff
@@ -2555,11 +2555,11 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UHV_RUS2</Tag>
-		<English>Be the first civilization to complete the Manhattan Project and the Apollo Program</English>
+		<English>Be the first civilization to complete the Manhattan Project and the Lunar Landing</English>
```
Comment (dguenms) [inline Assets/XML/Text/Victory.xml]: I would say "a Lunar Landing" sounds better.

File: Assets/XML/Text/Victory.xml [hunk 2]
```diff
@@ -1539,19 +1539,19 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UHV_JAP1_EPIC</Tag>
```
Comment (dguenms) [inline Assets/XML/Text/Victory.xml:1541]: I think the Epic and Marathon entries can be removed entirely now that the turns are displayed programmatically.

File: Assets/XML/Text/Victory.xml [hunk 3]
```diff
@@ -4878,15 +4878,15 @@
 		<Tag>TXT_KEY_VICTORY_RUSSIA_CONTROL_SIBERIA</Tag>
 		<English>Cities in Siberia: %d1 / %d2</English>
 		<French>Villes en Sib&#233;rie : %d1 / %d2</French>
-		<German>Cities in Siberia: %d1 / %d2</German>
+		<German>Staedte in Sibirien: %d1 / %d2</German>
```
Comment (dguenms) [inline Assets/XML/Text/Victory.xml]: Städte (with properly HTML escaped unicode).

File: Assets/XML/Text/Victory.xml [hunk 4]
```diff
@@ -5094,7 +5094,7 @@
 		<Tag>TXT_KEY_VICTORY_TRANSSIBERIAN_RAILWAY</Tag>
 		<English>Railroad from Moscow to a Siberian port</English>
 		<French>Chemin de fer de Moscou &#224; un port sib&#233;rien</French>
-		<German>Railroad from Moscow to a Siberian port</German>
+		<German>Eisenbahn von Moskau an einen sibirischen Hafen</German>
```
Comment (dguenms) [inline Assets/XML/Text/Victory.xml]: zu einem

**Review by dguenms — CHANGES_REQUESTED** [review]

File: Assets/XML/Text/Victory.xml (hunk 2 above)
Comment (Imperator-Knoedel) [reply Assets/XML/Text/Victory.xml:1541]: But you still have different culture values you need to achieve on each speed?

File: Assets/XML/Text/Victory.xml (hunk 4 above)
Comment (Imperator-Knoedel) [reply Assets/XML/Text/Victory.xml]: Grammar Nazis get the bullet too.

File: Assets/XML/Text/Victory.xml (hunk 2 above)
Comment (dguenms) [reply Assets/XML/Text/Victory.xml:1541]: Oh right, forget about this.

File: Assets/XML/Text/Victory.xml (hunk 4 above)
Comment (dguenms) [reply Assets/XML/Text/Victory.xml]: Grammar is fine, I just want a different preposition.

**Review by dguenms — APPROVED** [review]

**Pull request merged**